demos/image_classification_with_string_output/README.md

```diff
@@ -0,0 +1,74 @@
+# String output model demo {#ovms_string_output_model_demo}
+## Overview
+
+This demo demonstrates example deployment of a model with output precision `ov::element::string`. The output text is serialized into corresponding fields in gRPC proto/REST body. This allows the client to consume the text directly and avoid the process of label mapping or detokenization.
+
+### Download and prepare Image Net model using Keras and TensorFlow
+
+```bash
+pip install -r requirements.txt
+python3 download_model.py
```

Comment:
It might be handy to show the results of the command: model structure with tree and saved_model_cli or outout of model_metadata calls to ovms to show input and output format.